openapi-integration-tests/src/test/resources/ballerina_sources/project_openapi_info/service_file.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/http;
+import ballerina/openapi;
+
+@openapi:ServiceInfo {
```

Comment:
Is this a constant annotation, if not, is there a limitation if we use variables as the values?

Reply:
No, this is not a constant one, we can make it constant. Will change with separate PR by finding the impact of other usage.